targets/TARGET_Silicon_Labs/TARGET_EFM32/emlib/src/em_system.c

```diff
@@ -61,29 +60,23 @@ void SYSTEM_ChipRevisionGet(SYSTEM_ChipRevision_TypeDef *rev)
   uint8_t tmp;
 
   EFM_ASSERT(rev);
-  
```

Comment:
@0xc0170  this is the problematic code

Reply:
Is it that SECURE_READ was used, and now being replaced by just direct access?

Reply:
Ah damn, forgot to cherrypick the uvisor-specific path to that file. Will fix.

Reply:
@stevew817 have you added the uvisor-specific commit?

Reply:
Yes, added just now.